include/networkit/graph/Graph.hpp

```diff
@@ -54,6 +56,16 @@ struct WeightedEdge : Edge {
     WeightedEdge(node u, node v, edgeweight w) : Edge(u, v), weight(w) {}
 };
 
+struct WeightedEdgeWithId : WeightedEdge {
+    edgeid eid;
+
+    // Needed by cython
```

Comment:
There seems to be no Cython interface for this currently, so this comment seems misleading.